src/Components/SigDetailsTable/NewSigDetailsTable.js

```diff
@@ -234,7 +230,7 @@ const NewSigDetailsTable = ({ ruleName, affectedCount, isEmptyAccount }) => {
   };
 
   useEffect(() => {
-    const rowBuilder = (data) =>
+    const NewMatchesTableRow = (data) =>
```

Comment:
this `rowBuilder` is not the same as the `RowBuilder` we had as the component. I think you accidentally did a replace here.